Directory.Build.props

```diff
@@ -156,6 +175,7 @@
   <Import Project="eng\ProjectReferences.props" />
   <Import Project="eng\targets\Cpp.Common.props"  Condition="'$(MSBuildProjectExtension)' == '.vcxproj'" />
   <Import Project="eng\targets\CSharp.Common.props"  Condition="'$(MSBuildProjectExtension)' == '.csproj'" />
+  <Import Project="eng\targets\FSharp.Common.props"  Condition="'$(MSBuildProjectExtension)' == '.fsproj'" />
```

Comment:
Why bother import an empty file?

Reply:
will remove. Left over from earlier draft of this PR